ffi/mupdf.lua

```diff
@@ -370,9 +370,20 @@ function page_mt.__index:getSize(draw_context)
 
     M.fz_bound_page(context(), self.page, bounds)
     M.fz_transform_rect(bounds, ctm)
-    M.fz_round_rect(bbox, bounds)
 
+    -- NOTE: fz_bound_page returns an internal representation computed @ 72dpi...
```

Comment:
It was called fz_round_rect, not fz_bound_page , if you're talking about the thing you commented out.
(And that's as much as I'm able to review this :)

Reply:
Yep, that's correct ;).

`fz_round_rect` is just the geometry helper that rounds a rectangle with floating point coordinates to a rectangle with integer coordinates. the comment is indeed about where the rectangle with floating point coordinates comes from (`fz_bound_page`) and why we don't run `fz_round_rect` on it right now (because we do it in from post-scaling via `Geom:transformByScale`)